src/mono/wasm/build/WasmApp.targets

```diff
@@ -357,8 +355,6 @@
     <RemoveDir Directories="$(WasmAppDir)" />
     <WasmAppBuilder
       AppDir="$(WasmAppDir)"
-      MainJS="$(WasmMainJSPath)"
-      MainHTMLPath="$(WasmMainHTMLPath)"
```

Comment:
For backward compatibility, add `$(WasmMainHTMLPath)` to `WasmExtraFilesToDeploy`, if set.